python/tests/test_kmeans.py

```diff
@@ -63,14 +63,52 @@ def assert_centers_equal(
         assert a_center == pytest.approx(b_center, tolerance)
 
 
-def test_params() -> None:
+@pytest.mark.parametrize("default_params", [True, False])
+def test_params(default_params: bool) -> None:
     from cuml import KMeans as CumlKMeans
 
+    spark_params = {
+        "k": 2,
```

Comment:
For algos in mllib can we get these from Spark somehow vs hard coding?

Reply:
Good suggestion!